src/semgrep_agent/findings.py

```diff
@@ -38,13 +38,13 @@ class Finding:
     commit_date = attr.ib(
         type=Optional[datetime], default=None, hash=None, eq=False, kw_only=True
     )
-    metadata = attr.ib(
-        type=Optional[Mapping[str, Any]],
-        default=None,
-        hash=None,
-        eq=False,
-        kw_only=True,
-    )
+    metadata = attr.ib(type=Mapping[str, Any], hash=None, eq=False, kw_only=True,)
+
+    def is_blocking(self) -> bool:
+        """
+            Returns if this finding indicates it should block CI
+        """
+        return str(self.metadata.get("action", "block")) == "block"
```

Comment:
```suggestion
        return str(self.metadata.get("dev.semgrep.actions", "block")) == "block"
```

per https://github.com/returntocorp/semgrep-action/issues/34